lib/utils/getNextNonSharedLineCommentNode.js

```diff
@@ -0,0 +1,27 @@
+/* @flow */
+"use strict"
+
+const _ = require("lodash")
+
+function getNodeLine(node/*:: ?: postcss$node*/)/*: number | void*/ {
+  return _.get(node, "source.start.line")
+}
+
+module.exports = function getNextNonSharedLineCommentNode(node/*:: ?: postcss$node*/)/*: postcss$node | void*/ {
```

Comment:
I don't know if it matters, but `getPreviousNonSharedLineCommentNode` has slightly different formatting for this line.